diff -ur ./llvm/cmake/modules/AddLLVM.cmake ./llvm/cmake/modules/AddLLVM.cmake
--- ./llvm/cmake/modules/AddLLVM.cmake	2018-08-01 03:51:55.000000000 -0400
+++ ./llvm/cmake/modules/AddLLVM.cmake	2019-05-09 16:06:31.456119724 -0400
@@ -1598,26 +1598,6 @@
   if(NOT LLVM_CODESIGNING_IDENTITY)
     return()
   endif()
-
-  if(APPLE)
-    if(NOT CMAKE_CODESIGN)
-      set(CMAKE_CODESIGN xcrun codesign)
-    endif()
-    if(NOT CMAKE_CODESIGN_ALLOCATE)
-      execute_process(
-        COMMAND xcrun -f codesign_allocate
-        OUTPUT_STRIP_TRAILING_WHITESPACE
-        OUTPUT_VARIABLE CMAKE_CODESIGN_ALLOCATE
-      )
-    endif()
-    add_custom_command(
-      TARGET ${name} POST_BUILD
-      COMMAND ${CMAKE_COMMAND} -E
-              env CODESIGN_ALLOCATE=${CMAKE_CODESIGN_ALLOCATE}
-              ${CMAKE_CODESIGN} -s ${LLVM_CODESIGNING_IDENTITY}
-              $<TARGET_FILE:${name}>
-    )
-  endif()
 endfunction()
 
 function(llvm_setup_rpath name)
diff -ur ./llvm/cmake/platforms/iOS.cmake ./llvm/cmake/platforms/iOS.cmake
--- ./llvm/cmake/platforms/iOS.cmake	2019-05-10 12:37:01.688274821 -0400
+++ ./llvm/cmake/platforms/iOS.cmake	2019-05-09 16:08:27.949546474 -0400
@@ -80,23 +80,3 @@
   SET(CMAKE_LIBTOOL ${CMAKE_LIBTOOL_val} CACHE FILEPATH "Libtool")
   message(STATUS "Using libtool ${CMAKE_LIBTOOL}")
 ENDIF()
-
-IF(NOT CMAKE_CODESIGN)
-  execute_process(COMMAND xcrun -sdk ${CMAKE_OSX_SYSROOT} -find codesign
-   OUTPUT_VARIABLE CMAKE_CODESIGN_val
-   ERROR_QUIET
-   OUTPUT_STRIP_TRAILING_WHITESPACE)
-  SET(CMAKE_CODESIGN ${CMAKE_CODESIGN_val} CACHE FILEPATH "Codesign")
-  message(STATUS "Using codesign ${CMAKE_CODESIGN}")
-ENDIF()
-
-IF(NOT CMAKE_CODESIGN_ALLOCATE)
-  execute_process(
-    COMMAND xcrun -sdk ${CMAKE_OSX_SYSROOT} -find codesign_allocate
-    OUTPUT_VARIABLE CMAKE_CODESIGN_ALLOCATE_val
-    ERROR_QUIET
-    OUTPUT_STRIP_TRAILING_WHITESPACE)
-  SET(CMAKE_CODESIGN_ALLOCATE ${CMAKE_CODESIGN_ALLOCATE_val} CACHE
-      FILEPATH "Codesign_Allocate")
-  message(STATUS "Using codesign_allocate ${CMAKE_CODESIGN_ALLOCATE}")
-ENDIF()
diff -ur ./llvm/projects/compiler-rt/cmake/Modules/AddCompilerRT.cmake ./llvm/projects/compiler-rt/cmake/Modules/AddCompilerRT.cmake
--- ./llvm/projects/compiler-rt/cmake/Modules/AddCompilerRT.cmake	2018-07-10 09:00:17.000000000 -0400
+++ ./llvm/projects/compiler-rt/cmake/Modules/AddCompilerRT.cmake	2019-05-10 12:55:31.399913584 -0400
@@ -290,14 +290,6 @@
         set_target_properties(${libname} PROPERTIES IMPORT_PREFIX "")
         set_target_properties(${libname} PROPERTIES IMPORT_SUFFIX ".lib")
       endif()
-      if(APPLE)
-        # Ad-hoc sign the dylibs
-        add_custom_command(TARGET ${libname}
-          POST_BUILD  
-          COMMAND codesign --sign - $<TARGET_FILE:${libname}>
-          WORKING_DIRECTORY ${COMPILER_RT_LIBRARY_OUTPUT_DIR}
-        )
-      endif()
     endif()
     install(TARGETS ${libname}
       ARCHIVE DESTINATION ${install_dir_${libname}}
diff -ur ./llvm/projects/compiler-rt/test/sanitizer_common/ios_commands/iossim_compile.py ./llvm/projects/compiler-rt/test/sanitizer_common/ios_commands/iossim_compile.py
--- ./llvm/projects/compiler-rt/test/sanitizer_common/ios_commands/iossim_compile.py	2017-04-28 00:55:35.000000000 -0400
+++ ./llvm/projects/compiler-rt/test/sanitizer_common/ios_commands/iossim_compile.py	2019-05-09 16:07:55.482727106 -0400
@@ -26,7 +26,3 @@
 ret = subprocess.call(sys.argv[1:])
 if ret != 0:
     sys.exit(ret)
-
-# If we produce a dylib, ad-hoc sign it.
-if output_type in ['shared', 'dylib']:
-    ret = subprocess.call(["codesign", "-s", "-", output])
